plugins/hls-explicit-record-fields-plugin/src/Ide/Plugin/ExplicitFields.hs

```diff
@@ -319,11 +320,11 @@ collectNamesRule = defineNoDiagnostics mempty $ \CollectNames nfp -> runMaybeT $
 -- | Collects all 'Name's of a given source file, to be used
 -- in the variable usage analysis.
 getNames :: TcModuleResult -> UniqFM Name [Name]
-#if __GLASGOW_HASKELL__ < 910
+
 getNames (tmrRenamed -> (group,_,_,_))   = collectNames group
-#else
-getNames (tmrRenamed -> (group,_,_,_,_)) = collectNames group
-#endif
+
+
+
```

Comment:
Did some formatter mess you up?

Reply:
Yeah I noticed that but I forgot to push it before the PR 🫠